_data/changelogs/component-range-slider.yml

```diff
@@ -2,6 +2,14 @@ title: Range slider
 type: component
 changelogURL:
 items:
+- date: NNNN-NN-NN
+  summary: Removed redundant ARIA attributes.
+  summaryAdditional: This ultimately improved accessibility usage, particularly with screen readers.
+  isBreaking: true
```

Comment:
```suggestion
  summary: Removed redundant ARIA attributes to improve the screen reader experience.
  summaryAdditional: Update range markup to incorporate these updates.
  isBreaking: true
  affectsMarkup: true
```

Reply:
Made a small change to `summaryAdditional` because on first read, I didn't read it as **instructions**

> summaryAdditional: To incorporate these changes, update your range component markup.

Let me know what you think!